Source/NotificationBarConfiguration.swift

```diff
@@ -35,7 +35,7 @@ public class NotificationBarConfiguration {
         duration = 4.0
         font = UIFont.systemFont(ofSize: 16.0)
         textColor = .white
-        padding = 4.0
+        padding = 1.0
```

Comment:
What's the reason behind this change?

Reply:
The idea of padding has changed. This is now just bottom padding.

Reply:
Should we rename it to `bottomPadding` then?

Reply:
We should.